tests/spec/w3c/headers-spec.js

```diff
@@ -250,19 +250,89 @@ describe("W3C — Headers", function() {
       const doc = await makeRSDoc(ops);
       expect($("#subtitle", doc).length).toEqual(0);
     });
-    
+  
     it("uses existing h2#subtitle as subtitle", async () => {
       const ops = makeStandardOps();
       ops.body = "<h2 id='subtitle'>This is a pre existing subtitle <code>pass</code></h2>" + makeDefaultBody();
+    
+      const doc = await makeRSDoc(ops);
       
+      const subTitleElement = doc.querySelectorAll("h2#subtitle");
+      expect(subTitleElement.length).toEqual(1);
+    
+      const confSubTitle = doc.defaultView.respecConfig.subtitle;
+    
+      if (confSubTitle) {
```

Comment:
This can't be an `if` here. It must equal exactly one thing.

Reply:
I get it. This `if` would vary for each of the 3 tests that I have introduced. Correct ?

Reply:
Correct. If you need to split into multiple tests, that's totally ok.